### PR TITLE
Feature rain

### PR DESCRIPTION
**Pull request by KaushikShivam**
## It uses the MVC structure in a way that:
- ```Models``` folder contains Daily, Forcast, Search
- ```views``` folder contains forecastView, formView and a base file for all DOM elements
- ```index``` which is the controller file

## It uses technologies such as 
- Babel
- Webpack
- ESlint
- npm package managers

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ * - Search object
+ */
+const state = {
+  tempType: 'fahrenheit',
+};
+
+/**
+ * SEARCH CONTROLLER
+ */
+const searchController = async (query = 'London') => {
```
[the remark this answers is not in the document]
Comment (KaushikShivam) [reply src/js/index.js]: Resolved

File: src/js/models/Search.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-alert */
+import { KEY, BASE_URL } from '../config';
+
+export default class Search {
+  constructor(query) {
+    this.query = query;
+  }
+
+  async getResults() {
```
[the remark this answers is not in the document]
Comment (KaushikShivam) [reply src/js/models/Search.js]: resolved

File: src/js/config.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export const KEY = '98acf4edff3eebfa3ce173e73fc696cb';
+export const BASE_URL = 'http://api.openweathermap.org/data/2.5/forecast?';
```
Comment (zenott) [inline src/js/config.js]: It's better to use secure `https` url otherwise it will throw a mixed content error when served from a secure domain.

Comment (KaushikShivam) [reply src/js/config.js]: Thank you for the kind words and a great review. I will make the necessary changes as soon as I'm done with the final project.

**Pull request merged**